src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,274 @@
+
+        suite: TestSuite
+            The test suite where this test should get added to.
+        """
+
+        self.suite = suite
+        self.additionalSuites = additionalSuites
+        self.case = TestCase(self.name)
+
+        # Check if all test requirements (other tests) are fulfilled:
+        if not self.__checkTestRequirements(testResults):
+            printTester(f"Skipping test case '{self.name}' not all requirements ({str(self.requirements)}) are fulfilled")
```

Ruff

Use explicit conversion flag

Replace with conversion flag

(RUF010)